src/libraries/System.Net.Quic/src/Resources/Strings.resx

```diff
@@ -165,5 +165,13 @@
   <data name="net_ssl_app_protocols_invalid" xml:space="preserve">
     <value>The application protocol list is invalid.</value>
   </data>
+  <!-- shamelessly copied from System.Net.Sockets's resources, TODO: how to get around that? -->
```

Comment:
I would remove this comment. We already have few other places like this.

Reply:
yeah, at the time, I hoped reviewer would notice and suggest better way of doing stuff, then I found out we don't have a better way but forgot to remove the comment O:)